activerecord/lib/active_record/connection_adapters/sqlite3/schema_statements.rb

```diff
@@ -2,6 +2,41 @@ module ActiveRecord
   module ConnectionAdapters
     module SQLite3
       module SchemaStatements # :nodoc:
+        # Returns an array of indexes for the given table.
```

Comment:
Should I separate a commit/PR that refactoring `indexes` (changing `IndexDefinition`, consistent placed, and tiny tweaks) in this commit/PR?